app/backend/aquifers/views.py

```diff
@@ -161,7 +166,9 @@ def get_serializer_class(self):
         if self.request.method == 'GET':
             return serializers.AquiferSerializer
         else:
-            return serializers.AquiferDetailSerializer
+            if self.request.version == 'v1':
+                return serializers.AquiferDetailSerializerV1
+            return serializers_v2.AquiferDetailSerializerV2
```

Comment:
This looks fine, but is it possible to organize this better? I know there's currently no `v1/` folder, but how complex would it be to put all the new code in a new folder called `v2/`? So all new serializers, models, etc would reside there. Thoughts?

Reply:
Since it's just the serializers file that needs a new v2 file I don't think that's necessary yet, but we should definitely be prepared to re-organize things if the v2 serializers get messy.  This might start happening if we need different view methods for v2/v1.